Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py

```diff
@@ -161,106 +161,115 @@ def test_module_command(client: Client, *_) -> Tuple[None, None, str]:
 
 
 @logger
-def ip_command(client: Client, ip_address: str, ip_version: str) -> Tuple[str, Dict, Dict]:
+def ip_command(client: Client, ip_address: str, ip_version: str) -> Tuple[str, Dict, Union[Dict, list]]:
     """ Enrichment for IPv4/IPv6
 
     Args:
         client: Client object with request
-        ip_address: ip address
+        ip_address: ip_ec address
         ip_version: IPv4 or IPv6
 
     Returns:
         Outputs
     """
-    raw_response = client.query(section=ip_version,
-                                argument=ip_address)
-    if raw_response:
-        title = f'{INTEGRATION_NAME} - Results for {ip_version} query'
-        context_entry: dict = {
-            'IP': {
+    query_args: list = argToList(ip_address)
+    raws: list = []
+    title = f'{INTEGRATION_NAME} - Results for ips query'
+    ip_ec: list = []
+    alienvault_ec: list = []
+    dbotscore_ec: list = []
+    for arg in query_args:
+        raw_response = client.query(section=ip_version,
+                                    argument=arg)
+        if raw_response:
+            raws.append(raw_response)
+            ip_ec.append({
                 'Address': raw_response.get('indicator'),
                 'ASN': raw_response.get('asn'),
                 'Geo': {
                     'Country': raw_response.get('country_code'),
                     'Location': f'{raw_response.get("latitude")},{raw_response.get("longitude")}'
                 }
-            },
-            'AlienVaultOTX': {
+            })
+            alienvault_ec.append({
                 'IP': {
                     'Reputation': raw_response.get('reputation'),
-                    'IP': ip_address
+                    'IP': arg
                 }
-            },
-            'DBotScore': {
+            })
+            dbotscore_ec.append({
                 'Indicator': raw_response.get('indicator'),
                 'Score': calculate_dbot_score(raw_response.get('pulse_info', {})),
-                'Type': ip_version,
+                'Type': arg,
                 'Vendor': 'AlienVault OTX v2'
-            }
-        }
-        context: dict = {
-            f'IP({outputPaths.get("ip")})': context_entry.get('IP'),
-            f'DBotScore({outputPaths.get("dbotscore")})': context_entry.get('DBotScore'),
-            f'AlienVaultOTX.IP(val.IP && val.IP === obj.IP)': context_entry.get('AlienVaultOTX', {}).get('IP')
-        }
+            })
+        else:
+            return f'{INTEGRATION_NAME} - Could not find any results for given query', {}, {}
```

Comment:
replace the return, it will cause inconsistency in the results, for example:
assuming 1.1.1.1 will return a result and 1.1.1.2 will not,
`['1.1.1.1', '1.1.1.2']` will have different behavior than `['1.1.1.2', '1.1.1.1']`

same for all

Reply:
I checked if there are any raws at all from that query, is it ok? or it's not sufficient?
Thanks